docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,130 @@
+Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS library, that is a replacement of the current styling engine that uses emotion/styled-components under the hood.
+We also host Design Kits and pre-built Templates.
+
+Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.
+
+We're also making ambitious moves to incorporate our full suite of components into Toolpad, a low-code admin builder tool for assembling full-stack apps faster than ever.
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).
+
+## The culture
+
+MUI is a fully remote company with a team that spans the globe.
+The majority of our work is asynchronous, and we rely on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
+
+For additional details about the culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+We believe CSS-in-JS is the best styling API with the best developer experience for writing CSS.
+We've used it in Material UI since v1 (back then, with JSS, and currently with Emotion in v5).
+However, there are key challenges we need to overcome to keep using this stack: improving performance and compatibility with React 19 features, including React Server Components.
+That's why, about a year ago, we started to work on Pigment CSS!
+We recently released an early alpha version, [introduced at React Conf 2024](https://mui.com/blog/introducing-pigment-css/), for which we're seeing [early positive adoption](https://npm-stat.com/charts.html?package=@stylexjs/stylex,@pigment-css/react) and [some community validation](https://x.com/JoshWComeau/status/1767224693575053692).
+
+This is just the beginning, though.
+There's so much to do to make Pigment CSS a reliable and mature option for the future of Material UI and for many developers looking for a next-generation CSS-in-JS styling engine.
+We're well-positioned to make this happen, so we're looking for a talented engineer to join the team and help us get further beyond!
+
+### Why this is interesting
+
+The core components that will depend on this library are widely used, you will receive a lot of feedback on your work.
+The community has high expectations, they will push you to do better, every day.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
```

Comment:
```suggestion

```

I'd cut this entire part; don't think it's adding anything new when compared to the "Why we're hiring" section. That section, together with the role's key responsibilities (+ best/worse parts, if we're keeping that), should already give a pretty thorough view of "why this is interesting".

Reply:
Fair enough 👌